pkg/controller/bookkeepercluster/bookkeepercluster_controller.go

```diff
@@ -265,7 +265,14 @@ func (r *ReconcileBookkeeperCluster) reconcileFinalizers(bk *bookkeeperv1alpha1.
 				return fmt.Errorf("failed to update Bookkeeper object (%s): %v", bk.Name, err)
 			}
 			if err = r.cleanUpZookeeperMeta(bk, pravegaClusterName); err != nil {
-				return fmt.Errorf("failed to clean up metadata (%s): %v", bk.Name, err)
+				// emit an event for zk metadata cleanup failure
+				message := fmt.Sprintf("failed to clean up metadata (%s): %v", bk.Name, err)
```

Comment:
Lets clearly state that we failed to cleanup metadata from **zookeeper**  preferably also mention the znode path  "/pravega/<pravega-cluster-name>". So users know what they need to delete manually.